sn_client/src/api.rs

```diff
@@ -143,13 +146,19 @@ impl Client {
             }
             match client_events_rx.recv().await {
                 Ok(ClientEvent::ConnectedToNetwork) => {
-                    info!("Client connected to the Network.");
+                    is_connected = true;
+                    info!("Client connected to the Network {is_connected:?}.");
+                    println!("Client connected to the Network");
```

Comment:
I think we already have the progress bar that prints to the console when the client is connected?